src/app/zap-templates/zcl/operational-credentials-cluster.xml

```diff
@@ -38,11 +38,13 @@ fabric-scoped data.</description>
       <arg name="VendorId" type="INT16U"/>
     </command>
 
-    <command source="client" code="0x00" name="GetFabricId" optional="false">
-      <description>Retrieve the fabricID of the admin executing this command.</description>
+    // TODO: Remove SetFabric + SetFabricResponseonce addOptCert command is implemented
+    <command source="client" code="0x00" name="SetFabric" optional="false">
+      <description>Sets the vendorID of the fabric.</description>
+      <arg name="VendorId" type="INT16U"/>
     </command>
 
-    <command source="server" code="0x01" name="GetFabricIdResponse" optional="false">
+    <command source="server" code="0x01" name="SetFabricResponse" optional="false">
       <description>Sender expects GetFabricIdResponse command if the GetFabricId command was correctly received</description>
       <arg name="FabricId" type="FABRIC_ID"/>
```

Comment:
Because this is just temporary. Here is the flow since addOptCert is not implemented yet:
1) Need something on incoming messages and commands to identify a user, the commissioner + device node id are a good tuple value that would be unique per fabric
2) When pairing finishes, we set the fabricId of the new admin to be the commissioner node id and we also store the device node id as node id
3) When a command comes in, we retrieve the commissioner node id to find the correct admin so that we can update the fabric label (and in this case set the vendor id)
If I were to let the commissioner node id, when I get the in cluster command handling, I wouldn't be able to recognise which in the list of admins is the admin we should set this fabric id on (since it would only contain an admin id + a device node id). In the interim of addOptCert + fabricIndex being implemented where I'll be able to find my admin properly, this was an easy way of doing it.

Reply:
OK.  All of this needs to be _very_ clearly documented somewhere in the code. Because right now the code is doing things that seem to not make any sense (and that would in fact be prevented by a sane type system.....)